tools/ansible/roles/dockerfile/templates/Dockerfile.j2

```diff
@@ -46,6 +46,12 @@ RUN dnf -y update && dnf install -y 'dnf-command(config-manager)' && \
 
 RUN pip3 install virtualenv build
 
+RUN dnf -y update && dnf install -y wget && \
+    wget https://static.rust-lang.org/dist/rust-1.41.0-powerpc64le-unknown-linux-gnu.tar.gz && \
```

Comment:
- Why is this needed?
- Why are you hardcoding ppc64le?

Reply:
The rust compiler is required to install cryptography package on ppc64le.

Reply:
I just noticed the `if [ $(uname -m) = ppc64le ]` part. Can we rework this a bit?

Unless I'm missing something, `tini_architecture` and `kubectl_architecture` look exactly the same. Maybe we can combine them into a single `image_architecture` variable? You can then use that variable in a jinja conditional in Dockerfile.j2. See that file for other examples of conditionals.

Reply:
I have made the changes, Please review.

Reply:
@shanemcd Please review.

Reply:
Nice. Thank you.